apps/financial-aid/web-veita/src/routes/Municipalities/municipalities.tsx

```diff
@@ -121,6 +133,17 @@ export const Municipalities = () => {
           þessu upplýsingum?
         </div>
       )}
+
+      <NewMunicipalityModal
+        isVisible={isModalVisible}
+        setIsVisible={(visible) => {
+          setIsModalVisible(visible)
+        }}
+        activeMunicipality={municipalities
```

Comment:
This should be named in plural right?

And should it be activeMunicipalitiesCodes?